### PR TITLE
#14: Adds basic 'tell' command.

### LINKED ISSUE #14: 'tell' command
Implement a command that allows players to sent private messages between one another.

### PR DESCRIPTION
**Pull request by fado**
I messed up the branch name... or I started out intending the write the who command and ended up writing the tell command instead....

### CODE REVIEW

File: lib/command.py [hunk 1]
```diff
@@ -40,6 +41,19 @@ def say(self, player: Player, message: str):
                 # send them a message telling them what the player said
                 other_player.message(f"{player.name} says: {message}")
 
+    def tell(self, player: Player, params):
+        """ tell <player> message - Whisper a private message to the target player."""
+        target = params.split(' ')[0]
+        message = ' '.join(params.split(' ')[1:])
```
Comment (fichilleen) [inline lib/command.py:47]: This will throw an exception if the syntax is incorrect (ie we just get the command `tell`)

File: lib/command.py [hunk 2]
```diff
@@ -40,6 +41,19 @@ def say(self, player: Player, message: str):
...
+
+        if target.lower() not in [player.name.lower() for player in self.game_state.list_other_players(player)]:
```
Comment (fichilleen) [inline lib/command.py:49]: If we want to keep working on this - the `lower` thing is smart, but it's expensive to recompute on commands - we might be better indexing by lower case

**Review by fichilleen — APPROVED** [review]

File: lib/command.py (hunk 1 above)
Comment (fado) [reply lib/command.py:47]: It just returns 'There is nobody online by that name.' in that case.

File: lib/command.py (hunk 2 above)
Comment (fado) [reply lib/command.py:49]: Not sure what you mean by indexing here.

File: lib/command.py (hunk 1 above)
Comment (fichilleen) [reply lib/command.py:47]: How come it's not an `IndexError` since there wouldn't be a second index on the split?

File: lib/command.py (hunk 2 above)
Comment (fado) [reply lib/command.py:49]: Oh, I think I understand.

Comment (fichilleen) [reply lib/command.py:49]: Sorry, I suppose I should say, have the key lowercase

File: lib/command.py (hunk 1 above)
Comment (fado) [reply lib/command.py:47]: At a first glance, I'm not entirely sure. :P

**Pull request merged**